Software/scan_key/AuthenticationSQL.py

```diff
@@ -24,19 +24,21 @@ def machineAuth(id, cursor):
     for auth in authdata:
         if(auth[machineType] != 1):
             print ("USER IS NOT AUTHORIZED\n")
+            os.system("omxplayer deny.wav &")
```

Comment:
An aural feedback class would do the trick - just have a deny function that calls the appropriate soundtrack.